scroll-animations-1/Overview.bs

```diff
@@ -487,21 +492,37 @@ The procedure to <dfn>resolve a container-based offset</dfn> given
     first matching condition from the following:
     <div class="switch">
 
-    :   <var>offset</var> is <code>auto</code>
+    :   If <var>offset</var> is <code>auto</code>:
     ::  Either the beginning or the ending of {{source}}'s scroll range in
         {{orientation}} depending on whether the offset is {{start}} or {{end}}.
 
-    :   <var>offset</var> is a <<length-percentage>>
+    :   If <var>offset</var> is a {{CSSNumberish}} and matches <<number>>:
+    ::  The distance indicated by the value in <a value>pixel unit</a> along
+        {{source}}'s scroll range in {{orientation}}.
+
+
+    :   If <var>offset</var> is a {{CSSNumberish}} and matches
```

Comment:
Can we link "matches" to https://drafts.css-houdini.org/css-typed-om-1/#cssstylevalue-match-a-grammar ?

Reply:
Done.